app.R

```diff
@@ -77,6 +77,15 @@ plot_chart <- function(chart_data, col, title, show_legend=FALSE) {
 #' @examples
 #' plot_map(map_data, "World Map")
 plot_map <- function(map_data, title, casetype='confirmed') {
```

Comment:
If you can deduce title from casetype, just remove title from the parameter list. Remember to remove it in all function calls

In addition, you need to include whether it is "Per 1M". Try to pass a boolean to indicate that. And update the title accordingly

Reply:
added "Per 1 M" in the legend. Did not remove the title as it was giving some errors in the line plot function. Overall code is working fine